beacon-chain/core/epoch/spectest/epoch_processing_crosslink_test.go

```diff
@@ -0,0 +1,63 @@
+package spectest
+
+import (
+	"io/ioutil"
+	"reflect"
+	"testing"
+
+	"github.com/ghodss/yaml"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/epoch"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/params/spectest"
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+)
+
+func runCrosslinkProcessingTests(t *testing.T, filename string) {
+	file, err := ioutil.ReadFile(filename)
```

Comment:
Can the logic from line 17-25 be abstracted in some common method?

Reply:
will address this in a separate PR, better to do it all at once with the outstanding spec testing PRs.